ls1intum/Artemis
# `Programming exercises`: Fix test case issue indicator for task descriptions with brackets

### Checklist
#### General
#### Client

### Motivation and Context
When the title of a task cotains brackets (for example because it meantions a method that the student has to implement), these brackets get mistaken as part of the test methods.

### Description
I updated the Regex

### Steps for Testing
Prerequisites:
- 1 Instructor

1. Edit / Create a Programming Exercise
2. Add a new task with brackets inside, e.g. `[task][Implement myMethod()](testMyMethod)`
3. Check that no test case issues are present (if the test acutally exsts) or the test case name gets parsed correctly (if the test does not exist)

### Review Progress





#### Code Review
#### Manual Tests


### Screenshots
Issue on Production:
![grafik](https://user-images.githubusercontent.com/26540346/165170443-515935ba-405c-45c0-81ce-d02f5d1e8a61.png)

Correctly parsed name if the test actually does not exist:
![grafik](https://user-images.githubusercontent.com/26540346/165170549-284f0c9c-b799-428f-8db3-205a0c9d4aab.png)
